scripts/tests/gaia_tests.bats

```diff
@@ -139,6 +139,20 @@ setup() {
   assert_equal "$str1_statom_diff" "1000"
 }
 
+@test "[INTEGRATION-BASIC-GAIA] packet forwarding automatically liquid stake" {
```

Comment:
Integration test is written as simple as possible since rest of them are checked on previous cases.